webapp/app.py

```diff
@@ -96,13 +96,16 @@ def dashboard_index(path=None):
 
     @app.route("/config.js")
     @app.route("/manifest.json")
+    @app.route("/version.json")
     @app.route("/ghost-bundle.svg")
     def dashboard_files():
         """
         Load dashboard files directly
         """
-
-        return flask.render_template("dashboard" + flask.request.path)
+        return flask.send_from_directory(
```

Comment:
`render_template` is return the MIME type HTML which is causing the dashboard to fail, whereas `send_from_directory` is resolving the proper MIME type.